Loop/Managers/LoopDataManager.swift

```diff
@@ -707,6 +710,7 @@ extension LoopDataManager {
             self.logger.default("Loop running")
             NotificationCenter.default.post(name: .LoopRunning, object: self)
 
+            self.checkAlerts()
```

Comment:
Why not have something that listens to `.LoopRunning` (above)?

Reply:
see my other response